src/applications/vaos/tests/list/appointment-list.unit.spec.jsx

```diff
@@ -192,17 +214,40 @@ describe('VAOS integration: appointment list', () => {
     mockAppointmentInfo({
       requests: [request],
     });
+    const now = moment().utcOffset('-06:00');
+    mockParentSites(['983'], [parentSite983]);
+    mockSupportedFacilities({
+      siteId: 983,
+      parentId: 983,
+      typeOfCareId: 'CR1',
+      data: [
+        {
+          attributes: {
+            expressTimes: {
+              start: '00:00',
+              end: '23:59',
+              timezone: 'MDT',
+              offsetUtc: '-06:00',
+            },
+          },
+        },
+      ],
+    });
     const initialStateWithExpressCare = {
       featureToggles: {
         ...initialState.featureToggles,
         vaOnlineSchedulingExpressCare: true,
       },
+      appointment: {
```

Comment:
Do you need this `appointment` object for the test to run? It should use the default state set by the appointments reducer with the way we've set up this test.